src/devices/ExplorerHat/README.md

```diff
@@ -0,0 +1,20 @@
+﻿# Explorer HAT Pro (Pimoroni)
+
+## Summary
+
+[Explorer HAT Pro](https://shop.pimoroni.com/products/explorer-hat) is an add-on board for Raspberry Pi.
+
+![Explorer HAT Pro](https://res.cloudinary.com/practicaldev/image/fetch/s--h3gOjXAH--/c_limit%2Cf_auto%2Cfl_progressive%2Cq_auto%2Cw_880/https://user-images.githubusercontent.com/10654401/63101233-e88c4b80-bf78-11e9-87ff-20e7a2809c40.png)
```

Comment:
please upload this picture in here instead if possible (if license forbids then I'd recommend just making any picture), otherwise I'd remove the picture since this can be used by 3rd parties to track who opened this readme page which I doubt most of the users would like (I wouldn't)

Reply:
I take the photo myself. It's my own rpi + explorer hat devices

Reply:
Can you at least use just `https://user-images.githubusercontent.com/10654401/63101233-e88c4b80-bf78-11e9-87ff-20e7a2809c40.png` instead of the cloudinary proxy server? (I'm not getting reference to cloudinary when I upload the same way)

Reply:
Yes, of course